.github/actions/test-dotnet/action.yml

```diff
@@ -13,12 +13,32 @@ inputs:
     required: false
     default: quiet
     description: Verbosity parameter for dotnet test.
+  ui-test-parallelism:
+    required: false
+    default: "-1"
+    description: >
+      Determines how many UI tests will run in parallel. Affects both xUnit's maxParallelThreads configuration and the
+      MaxParallelTests configuration of Lombiq UI Testing Toolbox. If set to "-1" then the configuration coming from the
+      code won't be overridden.
 
 runs:
   using: "composite"
   steps:
+    - name: Set UI Test Parallelization
+      shell: pwsh
+      run: |
+        if (${{ inputs.ui-test-parallelism }} -eq -1)
+        {
+            Write-Output "The UI test parallelism configuration isn't overridden."
+            Exit 0
+        }
+
+        "Lombiq_Tests_UI__OrchardCoreUITestExecutorConfiguration__MaxParallelTests=${{ inputs.ui-test-parallelism }}" >> $Env:GITHUB_ENV
+        $Env:Lombiq_Tests_UI__OrchardCoreUITestExecutorConfiguration__MaxParallelTests = "${{ inputs.ui-test-parallelism }}"
 
-    - name: Tests
+        ${{ github.action_path }}/replace-xunit-maxParallelThreads.ps1 -MaxParallelThreads ${{ inputs.ui-test-parallelism }}
```

Comment:
Why are you using a mixed case file name? I mean this is both camelCase and kebab-case, which means it's
![image](https://user-images.githubusercontent.com/4007293/176494687-526f54cc-b86e-4411-8f39-dcd22a5cee18.png)

Also is there any reason why we aren't naming the ps1 files cmdlet-style? I think in this case that would be `Set-XUnitMaxParallelThreads` if we have [use the approved verbs](https://docs.microsoft.com/en-us/powershell/scripting/developer/cmdlet/approved-verbs-for-windows-powershell-commands?view=powershell-7.2#:~:text=replaces%20data%20on%20an%20existing%20resource).

Reply:
:D Kebab is for the script name, and I wanted to reference the xUnit config exactly for the rest. Since this is not a cmdlet (nor a function), just a simple script, I wouldn't make its file name look like one's. I renamed the script not to mix cases, though.

Reply:
What I see is that PSScriptAnalyzer shows a PSUseApprovedVerbs warning on functions regardless whether they are simple scripts or cmdlet.
![image](https://user-images.githubusercontent.com/4007293/176560611-8be4bc16-c9fc-414b-b499-9c584cabef9d.png)

If you have to apply the cmdlet style to functions, then why wouldn't you apply it to script file names as well? You invoke either of them the same way and it's trivial to convert one to the other so I'm not sure why should their naming be different.

Alternatively we can throw the cmdlet style naming to the garbage and blacklist the related rules in the PSScriptAnalyzer config. That also works for me, but it should be consistent.

Reply:
If you include the action directory in the PATH then you can easily call a function like that, the difference vanishes.
Anyway, I'm fine with tabling this for later - if you want it done everywhere at once, then I will do the renaming (and the PATH trick) in OSOE-150.